packages/react/src/components/Masthead/MastheadL1.js

```diff
@@ -20,19 +21,31 @@ const { prefix } = settings;
  * @typedef {object} navigation Object containing navigation elements
  * @returns {*} Masthead component
  */
-const MastheadL1 = () => {
+const MastheadL1 = ({ isShort }) => {
   const className = cx({
     [`${prefix}--masthead__l1`]: true,
+    [`${prefix}--masthead__l1--short`]: isShort,
+  });
+
+  const nameShortL1 = cx({
+    [`${prefix}--masthead__l1-name--shortl1`]: isShort,
+  });
+  const eyebrowShortL1 = cx({
+    [`${prefix}--masthead__l1-name-eyebrow--shortl1`]: isShort,
+  });
+  const titleShortL1 = cx({
+    [`${prefix}--masthead__l1-name-title--shortl1`]: isShort,
   });
 
   return (
     <div className={className}>
-      <div className={`${prefix}--masthead__l1-name`}>
-        <span className={`${prefix}--masthead__l1-name-eyebrow`}>
+      <div className={`${prefix}--masthead__l1-name ${nameShortL1}`}>
+        <span
+          className={`${prefix}--masthead__l1-name-eyebrow ${eyebrowShortL1}`}>
           <ArrowLeft16 />
           <a href="#">Eyebrow</a>
         </span>
-        <span className={`${prefix}--masthead__l1-name-title`}>
+        <span className={`${prefix}--masthead__l1-name-title ${titleShortL1}`}>
           Stock Charts
```

Comment:
also pass this in through a prop instead

Reply:
done